traefik/values.yaml

```diff
@@ -72,6 +72,127 @@ volumes: []
 #   mountPath: "/config"
 #   type: configMap
 
+# Port Redirections
+# Added in 2.2, you can make permanent redirects via entrypoints.
+# https://docs.traefik.io/routing/entrypoints/#redirection
+redirections:
+  web:
+    enabled: false
+    to: websecure
+    scheme: https
+    permanent: true
+    priority: 1
+
+# Logs
+# https://docs.traefik.io/observability/logs/
+logs:
+  # Traefik logs concern everything that happens to Traefik itself (startup, configuration, events, shutdown, and so on).
+  general:
+    # By default, the logs use a text format (common), but you can
+    # also ask for the json format in the format option
+    # format: json
+    # By default, the level is set to ERROR. Alternative logging levels are DEBUG, PANIC, FATAL, ERROR, WARN, and INFO.
+    level: ERROR
+  access:
+    # To enable access logs
+    enabled: false
+    # By default, logs are written using the Common Log Format (CLF).
+    # To write logs in JSON, use json in the format option.
+    # If the given format is unsupported, the default (CLF) is used instead.
+    # format: json
+    # To write the logs in an asynchronous fashion, specify a bufferingSize option.
+    # This option represents the number of log lines Traefik will keep in memory before writing
+    # them to the selected output. In some cases, this option can greatly help performances.
+    # bufferingSize: 100
+    # Filtering https://docs.traefik.io/observability/access-logs/#filtering
+    filters: {}
+      # statuscodes: "200,300-302"
+      # retryattempts: true
+      # minduration: 10ms
+    # Fields
+    # https://docs.traefik.io/observability/access-logs/#limiting-the-fieldsincluding-headers
+    fields:
+      general:
+        defaultmode: keep
+        names: {}
+          # Examples:
+          # ClientUsername: drop
+      headers:
+        defaultmode: drop
+        names: {}
+          # Examples:
+          # User-Agent: redact
+          # Authorization: drop
+          # Content-Type: keep
+
+# Metrics
+# https://docs.traefik.io/v2.2/observability/metrics/overview/
+metrics:
```

Comment:
Can we have a seperate PR for the metrics block? Looks like we have different approaches of the implementation.

Reply:
Ive removed it in this PR and will open that one and a few others once this initial finishes up :)